Riot/Modules/Room/NotificationSettings/SwiftUI/RoomNotificationSettingsView.swift

```diff
@@ -25,16 +25,22 @@ struct RoomNotificationSettingsView: View {
     
     @State var notificationState: RoomNotificationState = RoomNotificationState.all
     
-    
-    var body: some View {
-        let leftButton = presentedModally ?
+    @ViewBuilder
+    var leftButton: some View {
+        if presentedModally {
             SwiftUI.Button(VectorL10n.cancel) {
                 viewModel.process(viewAction: .cancel)
             }
-            : nil
-        let rightButton = SwiftUI.Button(VectorL10n.save) {
+        }
+    }
+    
+    var rightButton: some View {
+        Button(VectorL10n.save) {
             viewModel.process(viewAction: .save)
         }
+    }
+    
```

Comment:
Moving views out of the body like above is a nice little help for readability. `leftButton` is marked as a ViewBuilder, because it may return the button or `EmptyView` (only necessary when the return type can change).

Reply:
Nice!